doc/multiBand.dox

```diff
@@ -0,0 +1,73 @@
+namespace lsst { namespace pipe { namespace tasks {
+/**
+\page pipeTasks_multiBand High-level Overview of Multi-Band Coadd Processing
+
+The goal of coadd measurement is to consistently detect and measure sources across coadds constructed in 
+multiple filter bands in a consistent and reasonable manner. Specifically, we would like to measure the same 
+(deblended) sources across all photometric bands. There are several advantages to imposing this requirement. 
+We get the same deblending in all the bands. This allows us to run forced photometry on consistently deblended 
+objects which gives us reasonable colors. It also permits us to obtain measurements of flux in the case of 
+drop-out objects such as Lyman-break galaxies. Our strategy for meeting the consistent deblending requirement 
+can be broken up into a number of simple steps.
+<DL> 
+  <DT> i. \ref DetectCoaddSourcesTask_ "DetectCoaddSourcesTask": Detect Sources in Individual Coadds</DT> 
+  <DD> This step is the last step in the process of constructing a coadd and only temporarily belongs in the 
+  Multi-Band Coadd processing pipeline. After scaling the variance plane in a coadd to correct for the 
+  effects of warping, sources are detected using the \ref SourceDetectionTask_ "SourceDetectionTask". This 
+  produces sources with footprints in each band. 
+  \par Inputs:
```

Comment:
Is it useful to list all the inputs and outputs etc on this overview page? I suspect what the casual reader is looking for is a description of the basic logic; if they need this level of detail, they can dive into the task documentation. Including it here just means there's another place to have to remember to update when things change!